src/components/NormalRightRow/index.jsx

```diff
@@ -1,19 +1,38 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import { shape, string } from 'prop-types';
+import Typography from '@material-ui/core/Typography';
 
 function NormalRightRow({ schema, classes }) {
-  const keywords = Object.keys(schema);
+  const skipKeywords = ['type', 'name', 'description', 'items', 'contains'];
```

Comment:
A short comment here about why we are skipping may be helpful for posterity. Same with the other skipKeywords variable in NormalLeftRow.

Reply:
Added comments!